queries/useConversationQuery.ts

```diff
@@ -1,15 +1,16 @@
-import { UseQueryOptions, useQuery } from "@tanstack/react-query";
+import { mutateObjectProperties } from "@/utils/mutate-object-properties";
+import { useQuery } from "@tanstack/react-query";
 import { getConversationByTopicByAccount } from "@utils/xmtpRN/conversations";
 import type { ConversationTopic } from "@xmtp/react-native-sdk";
 import { conversationQueryKey } from "./QueryKeys";
 import { queryClient } from "./queryClient";
-import { mutateObjectProperties } from "@/utils/mutate-object-properties";
 
 export type ConversationQueryData = Awaited<ReturnType<typeof getConversation>>;
 
 type IArgs = {
   account: string;
   topic: ConversationTopic;
+  context: string;
```

Comment:
I think context is too overloaded a term to be used just for logging. Other options could be

source
location
caller
 etc